src/Playwright.Tests/PageBasicTests.cs

```diff
@@ -204,8 +204,11 @@ public async Task ShouldWorkWithPageClose()
         [Fact(Timeout = TestConstants.DefaultTestTimeout)]
         public async Task ShouldFireLoadWhenExpected()
         {
+            var loadEvent = new TaskCompletionSource();
```

Comment:
Let's make the tests .NET 3.1 friendly. If we don't want to be 3.1 friendly let's remove the build step.

```suggestion
            var loadEvent = new TaskCompletionSource<bool>();
```

Reply:
We get the same in other places. Let's make a decision and apply it across the board.